components/harvest/actions/start-timer/start-timer.mjs

```diff
@@ -0,0 +1,29 @@
+import harvest from "../../harvest.app.mjs";
+
+export default {
+  key: "harvest-start-timer",
+  name: "Start Time Entry",
+  description: "Restart a stopped timer entry. [See docs here](https://help.getharvest.com/api-v2/timesheets-api/timesheets/time-entries/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    harvest,
+    timeEntryId: {
+      propDefinition: [
+        harvest,
+        "timeEntryId",
+        () => ({
+          isRunning: false,
+        }),
+      ],
+    },
+  },
+  async run({ $ }) {
+    const response = await this.harvest.restartTimeEntry({
+      $,
+      id: this.timeEntryId,
+    });
+    response && $.export("$summary", "Successfully started the time entry");
```

Comment:
Again, can we add another `$.export` when response is `undefined` or `null`?

Reply:
Response would always return an object on successful operation. Exception would be thrown if otherwise